src/test/java/codeu/model/store/basic/MessageStoreTest.java

```diff
@@ -61,6 +61,11 @@ public void testGetMessagesInConversation() {
     assertEquals(MESSAGE_TWO, resultMessages.get(1));
   }
 
+  @Test /*before adding message*/
```

Comment:
Comment should be added before @Test line.

Reply:
Or you can just add this to the function name. For example - testGetSizeBeforeAddingMessage(). That would be better i feel.